spec/models/conversation_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: conversations
+#
+#  id         :bigint           not null, primary key
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_conversations_on_name  (name) UNIQUE
+#
+require 'rails_helper'
+
+RSpec.describe Conversation, type: :model do
+  subject(:conversation) { FactoryBot.build(:conversation) }
+
+  describe 'validation test' do
+    context 'when we have required fields to fill in' do
```

Comment:
this title is wrong.
It says we have fields, but later name is nil.
It's confusing

Reply:
```
describe 'invalid conversation' do
  context 'when missing name attribute' do
    let(:nameless_conversation) { build(:conversation, :nameless) }
    it 'rises validation error' do
      expect(conversation.save!).to raise(ARError, "errror message")
    end
  end
end
```